README.md

```diff
@@ -480,6 +480,29 @@ And the charm's runtime will see `self.stored_State.foo` and `.baz` as expected.
 Also, you can run assertions on it on the output side the same as any other bit of state.
 
 
+# Emitted events
+If your charm deals with deferred events, custom events, and charm libs that in turn emit their own custom events, it can be hard to examine the resulting control flow.
+In these situations it can be useful to verify that, as a result of a given juju event triggering (say, 'start'), a specific chain of deferred and custom events is emitted on the charm. The resulting state, black-box as it is, gives little insight into how exactly it was obtained. One important source of information at debug time is the `debug-log`, which `scenario.utils.emitted_events` leverages to allow you to peep into that black box.
+
+Usage: 
+
+```python
+from scenario import State, DeferredEvent
+from scenario.utils import emitted_events
+state_out = State(deferred=[DeferredEvent('foo')]).trigger('start', ...)
+emitted = emitted_events(state_out)
+data = [(e.name, e.source) for e in emitted]
```

Comment:
Replying also to @benhoyt's analogous comment below. My rationale was as follows:
- a principled reason: the sequence of emitted events is not part of the charm's state, because in Scenario, State is 'what the charm can inspect or affect (via hook tools, pebble calls, or by inspecting the state db)', and what I'm adding here is something 'on top' of the State, that the charm is unaware of. So it kind of violates the boundaries of the State abstraction.
- a less principled reason: if it was on State, I'd expect its return type to be `Sequence[Event]`, or `Sequence[EventBase]`, which are hard to obtain, post-mortem, by parsing the juju-log.